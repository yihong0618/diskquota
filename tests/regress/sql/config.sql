--start_ignore
CREATE DATABASE diskquota;

\! gpconfig -c shared_preload_libraries -v $(./data/current_binary_name);
\! gpconfig -c diskquota.naptime -v 0 --skipvalidation
\! gpconfig -c max_worker_processes -v 20 --skipvalidation
\! gpconfig -c diskquota.hard_limit -v "off" --skipvalidation

\! gpstop -raf
--end_ignore

\c
-- Show the values of all GUC variables
-- start_ignore
SHOW diskquota.naptime;
-- end_ignore
SHOW diskquota.max_active_tables;
SHOW diskquota.worker_timeout;
SHOW diskquota.hard_limit;
