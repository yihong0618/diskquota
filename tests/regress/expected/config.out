\c
-- Show the values of all GUC variables
-- start_ignore
-- naptime cannot be 0 on Release build, so it will be 2
SHOW diskquota.naptime;
 diskquota.naptime 
-------------------
 0
(1 row)
-- end_ignore
SHOW diskquota.max_active_tables;
 diskquota.max_active_tables 
-----------------------------
 307200
(1 row)

SHOW diskquota.worker_timeout;
 diskquota.worker_timeout 
--------------------------
 60
(1 row)

SHOW diskquota.hard_limit;
 diskquota.hard_limit 
----------------------
 off
(1 row)

